### ISSUE TITLE
Mandibula cut plane cuts through the condyle

### ISSUE THREAD

**Issue by mrtig** [SlicerIGT/SlicerBoneReconstructionPlanner#27]
It's a simple two plane cut. I don't remember having this issue with 3 or 4 cut planes.
For some reason the cut extends two far back and also cuts off the right condyle.
![image](https://user-images.githubusercontent.com/12380462/108820738-4c0f2480-75c5-11eb-8ce1-9829ebe4b143.png)

**Comment by cmfsx** [comment]
yes, I could reproduce it. I never used two cuts only so the bug was never caught.

**Comment by mauigna06** [comment]
Hi @mrtig. Thank for using the module. 

> For some reason the cut extends two far back and also cuts off the right condyle.

This happens because the dynamicModeler takes the mandiblePlanes as they have infinite lenght and infinite width so it cuts through everything that is between them to make the resulting model, in this case the it cuts the right condyle as well.

The easier way to avoid this is changing the orientation of the mandiblePlanes so they don't involve part of the condyle:

![Screenshot5](https://user-images.githubusercontent.com/19158307/108852565-8926e700-75c4-11eb-8cf4-e4a3258434f8.png)

That the condyle may be cut doesn't alter the planning so it's a minor bug (thank you for reporting it anyway). 
@lassoan, do you have any idea on how to solve this?

**Comment by mrtig** [comment]
It's not a big issue, but changing the orientation of the plane is not a real solution.

**Comment by lassoan** [comment]
> That the condyle may be cut doesn't alter the planning so it's a minor bug (thank you for reporting it anyway).
> @lassoan, do you have any idea on how to solve this?

We added the option of specifying "Operation type" for exactly this purpose. You can add a sagittal cutting plane and set operation type so there cannot be any other cuts on the other side of the mandible.

**Comment by mauigna06** [comment]
Although it was fixed, how this works needs to be documented and clear for the user
